lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
I'm fine with any setting, just think that in code, our editors and ESLint
should match.

On Wed, Jul 27, 2016 at 11:58 AM, Henric Trotzig <notifications@github.com>
wrote:

> In lib/ImportStatements.js
> <https://github.com/Galooshi/import-js/pull/321#discussion_r72468184>:
>
> > @@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
> >  import Configuration from './Configuration';
>
> I'm about to change to 100 chars (from previously 80 I think), are we okay
> with that?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Galooshi/import-js/pull/321/files/5138c74aaed901105f9957cf22129203e9b5de70..7b1c648f01761d447fe1b5193caffe85d5d72636#r72468184>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH5TYoPB0Q1DmcKK33UkGo4nKURo1HMfks5qZ4AhgaJpZM4JU6-_>
> .
>